.yarn/patches/docusaurus-plugin-typedoc-api-npm-3.0.1-7dd061feb2.patch

```diff
@@ -0,0 +1,589 @@
+diff --git a/lib/index.d.ts.map b/lib/index.d.ts.map
+index c320d1ee3ebc178a4aa4317a97614bf4f1239982..380b88c4bc8e31bb8cd47aa6bb217523ad109958 100644
+--- a/lib/index.d.ts.map
++++ b/lib/index.d.ts.map
+@@ -1 +1 @@
+-{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../src/index.ts"],"names":[],"mappings":"AAOA,OAAO,KAAK,EAAE,WAAW,EAAE,MAAM,EAAe,MAAM,mBAAmB,CAAC;AAU1E,OAAO,KAAK,EAEX,iCAAiC,EACjC,aAAa,EAKb,MAAM,SAAS,CAAC;AAwCjB,MAAM,CAAC,OAAO,UAAU,gBAAgB,CACvC,OAAO,EAAE,WAAW,EACpB,aAAa,EAAE,iCAAiC,GAC9C,MAAM,CAAC,aAAa,CAAC,CAuVvB"}
+\ No newline at end of file
++{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../src/index.ts"],"names":[],"mappings":"AAOA,OAAO,KAAK,EAAE,WAAW,EAAE,MAAM,EAAe,MAAM,mBAAmB,CAAC;AAU1E,OAAO,KAAK,EAEX,iCAAiC,EACjC,aAAa,EAKb,MAAM,SAAS,CAAC;AA2CjB,MAAM,CAAC,OAAO,UAAU,gBAAgB,CACvC,OAAO,EAAE,WAAW,EACpB,aAAa,EAAE,iCAAiC,GAC9C,MAAM,CAAC,aAAa,CAAC,CA2VvB"}
+\ No newline at end of file
+diff --git a/lib/index.js b/lib/index.js
+index 57a9cb7ec34f4bc0201872853fcc278b1df6ee7c..c7c4bf8d537ecde4b7f1153fa7b4a1dbaff3c2a3 100644
+--- a/lib/index.js
++++ b/lib/index.js
+@@ -36,6 +36,7 @@ const DEFAULT_OPTIONS = {
+   onlyIncludeVersions: [],
+   packageJsonName: 'package.json',
+   packages: [],
++  packageCategories: [],
+   projectRoot: '.',
+   sortPackages: (a, d) => a.packageName.localeCompare(d.packageName),
+   sortSidebar: (a, d) => a.localeCompare(d),
+@@ -45,6 +46,8 @@ const DEFAULT_OPTIONS = {
+   routeBasePath: 'api',
+   tsconfigName: 'tsconfig.json',
+   typedocOptions: {},
++  remarkPlugins: [],
++  rehypePlugins: [],
```

Comment:
By any chance, can you try sending a PR upstream to get these changes accepted? I'd prefer if we could avoid maintaining permanent patches.

Reply:
Done. milesj/docusaurus-plugin-typedoc-api#123

Reply:
Thanks!